### PR TITLE
Donut charts would be misaligned after resizing the window.

### LINKED ISSUE #9863: Misalignment of text in donut charts
normal view (browser chrome, latest version of qt)
image

full screen view 
image

### PR DESCRIPTION
**Pull request by fniessink**
Fixes #9863.

### CODE REVIEW

**Review by wkoot — APPROVED** [review]

File: components/frontend/src/dashboard/MetricSummaryCard.js [hunk 1]
```diff
@@ -72,27 +72,10 @@ export function MetricSummaryCard({ header, onClick, selected, summary, maxY })
...
-            {
-                childName: "all",
-                target: "data",
-                eventHandlers: {
-                    onClick: () => {
-                        return [
-                            {
-                                target: "labels",
-                                mutation: () => {
-                                    return { active: false } // Clear tooltip before filtering
```
Comment (wkoot) [inline components/frontend/src/dashboard/MetricSummaryCard.js:85]: No longer needed?

File: components/frontend/src/dashboard/StatusPieChart.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ export function StatusPieChart({ animate, colors, events, height, label, maxY, s
     const data = STATUSES.map((status) => {
         const y = summary[STATUS_COLORS[status]]
         const yPercentage = Math.round((y / nrMetrics) * 100)
-        return { y: y, label: `${STATUS_NAME[status]}: ${nrMetricsLabel(y)} (${yPercentage}%)` }
+        return { y: y, label: `${STATUS_NAME[status]}\n${nrMetricsLabel(y)} (${yPercentage}%)` }
```
Comment (wkoot) [inline components/frontend/src/dashboard/StatusPieChart.js:24]: This seems very similar to the change in `StatusBarChart.js`, is the pattern used elsewhere?

File: components/frontend/src/dashboard/MetricSummaryCard.js (hunk 1 above)
Comment (fniessink) [reply components/frontend/src/dashboard/MetricSummaryCard.js:85]: Yes, this is made obsolete by not rendering the tooltip in the portal.

File: components/frontend/src/dashboard/StatusPieChart.js (hunk 2 above)
Comment (fniessink) [reply components/frontend/src/dashboard/StatusPieChart.js:24]: I don't think so.

**Pull request merged**